.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+{Description of the PR}
+
+Issues this may resolve (delete line if none):
+
+TODO:
+- [ ] Perform basic functions test on production robot
+- [ ] Test PR feature(s) on production robot
+- [ ] Perform basic functions test on prototype robot
```

Comment:
I feel performing tests on *two* robots is a bit much per PR and slows things down. I think just testing on production robot is fine, and I would be ok risking a regression on the prototype bot to speed things along. I'm a bit on the fence still though.